cpp/include/cuvs/neighbors/nn_descent.hpp

```diff
@@ -29,7 +29,7 @@
 
 namespace cuvs::neighbors::nn_descent {
 /**
- * @defgroup nn-descent The nn-descent algorithm.
+ * @defgroup nn_descent_cpp_index_params The nn-descent algorithm.
```

Comment:
nitpick:  this group isn't the algorithm 
```suggestion
 * @defgroup nn_descent_cpp_index_params The nn-descent algorithm parameters.
```